auth-web/src/components/auth/home/TestimonialQuotes.vue

```diff
@@ -9,27 +9,53 @@
   </section>
 </template>
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { defineComponent, onMounted, ref } from '@vue/composition-api'
+import vue from 'vue'
 
-@Component({})
-export default class TestimonialQuotes extends Vue {
-  private quoteIndex = 0
-  private quotes: Array<any> =
-    [
+export default defineComponent({
+  name: 'TestimonialQuotes',
+  setup () {
+    const quoteIndex = ref(0)
+    const quotes = [
       {
         text: 'We are really excited by the idea of incorporating as a Benefit Company, it really makes sense for us and the way we want to run our business.',
         author: 'Business Founder'
       }
-    ]
+    ] as { text: string, author: string}[]
 
-  mounted () {
-    setInterval(() => {
+    const setInterval = () => {
```

Comment:
this was a built in Window function I think? It's supposed to execute it every 5 seconds